software/gamevault.yml

```
@@ -0,0 +1,14 @@
+name: "GameVault"
+website_url: "https://gamevau.lt"
+source_code_url: "https://github.com/Phalcode/gamevault-backend"
+description: "GameVault is a gaming platform for organizing, downloading, and playing DRM-free games from your own server, complete with metadata enrichment and user-friendly interface."
+licenses:
+  - CC-BY-NC-SA-4.0
+platforms:
+  - Javascript
+  - Nodejs
+  - Docker
```

Js isnt needed as Nodejs is already present

```suggestion
platforms:
  - Nodejs
  - Docker
```